samples/httpbin/sample-client/fortio-deploy.yaml

```diff
@@ -6,6 +19,11 @@ spec:
   replicas: 1
   template:
     metadata:
+      annotations:
+        # This annotation causes Envoy to serve cluster.outbound statistics via 15000/stats
+        # in addition to the stats normally served by Istio.  The Circuit Breaking example task
+        # gives an example of inspecting Envoy stats.
+        sidecar.istio.io/statsInclusionPrefixes: cluster.outbound,cluster_manager,listener_manager,http_mixer_filter,tcp_mixer_filter,server,cluster.xds-grpc
```

Comment:
why this? this seems gratuitous

Reply:
PR https://github.com/istio/istio/pull/10184 turned off most Envoy stats collection.  `curl localhost:15000/stats` returns only a few things.  This was done to save memory.  Unfortunately it turns off all stats for normal outbound and listener which means that the Circuit Breaking instructions didn't work.  It also turns off collections about TLS problems on listeners!

I did https://github.com/istio/istio/pull/11398 which allows an annotation to turn collection back on.  By including `cluster.outbound` the Circuit Breaking instruction to do `kubectl exec -it $FORTIO_POD  -c istio-proxy  -- sh -c 'curl localhost:15000/stats' | grep httpbin | grep pending` works again.  I am writing up documentation this morning on how to use this annotation.